packages/pwa-kit-dev/src/utils/resolver-utils.test.js

```diff
@@ -8,28 +8,43 @@
 import path from 'path'
 import * as resolverUtils from './resolver-utils'
 
+const PROJECT_PATH_GOOD = '/home/user/testproject'
+const PROJECT_PATH_BAD = '/home/user/test.project'
```

Comment:
nitpicking, the word "bad" implies this is for a negative test, but it should be supported

```suggestion
const PROJECT_PATH_WITH_DOT = '/home/user/test.project'
```